cli/types/net-stubbing.ts

```diff
@@ -0,0 +1,553 @@
+// Copied from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/methods/index.d.ts
```

Comment:
How come we're copying instead of importing [@types/methods](https://www.npmjs.com/package/@types/methods)?

Reply:
It seems that this file somehow got into this PR when I was switching between branches. Removed it.